modules/apps/object/object-service/src/main/java/com/liferay/object/internal/sort/field/ObjectEntryFieldSortDSLQueryBuilderContributor.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.sort.field;
```

Comment:
I would put all the contributors in the same package to follow liferay conventions. And maybe the sorts in a different package.

Reply:
Done